### DISCUSSION TITLE
Does LMdeploy support latest version of flash attention, page attention?

### DISCUSSION THREAD

**Discussion by ywangwxd** [InternLM/lmdeploy#480] [Ideas]
They are new techniques for optimizing inference performance of LLM.

**Comment by lvhan028** [comment]
flash attention 2 is already integrated.
page attention is in impelmentation

**Reply by ywangwxd** [reply]
Great! Looking forward to the new release.  
Btw,  It will be better to introduce what features have been included in the release notice.
